pkg/generator/generator.go

```diff
@@ -266,6 +296,16 @@ func (g *Generator) renderTmp() error {
 	return renderCodegenFiles(cDir, g.repoPath, apiDirName(g.apiVersion), version(g.apiVersion), g.projectName)
 }
 
+func renderTmpGitignore(tDir string) error {
```

Comment:
I don't think this is necessary. Just let the project root `.gitignore` handle all the ignoring files.